### ISSUE TITLE
Language problem on FakeI18n

### ISSUE THREAD

**Issue by Nuxor3000** [chaskiq/chaskiq#606]
In the line i18n.locale = document.documentElement.lang || 'en' in app/javascript/src/shared/FakeI18n.ts

document.documentElement.lang is empty

I replaced it by :

const currentLang = navigator.language || navigator['userLanguage'];
i18n.locale = currentLang || 'en';

which set the correct language but when connecting or failing, language is back to english

Do you have a clue why ?

**Comment by michelson** [comment]
Hi @Nuxor3000, what user language do you have in your browser?

**Comment by Nuxor3000** [comment]
italian 
I modified the connect page to use the locales I added in it.yml
Seems like the language is set somewhere else also but I don't find where

**Comment by Nuxor3000** [comment]
Oh and that's occuring both in firefox and chrome in the same way (I didn't try others)

**Comment by Nuxor3000** [comment]
And I also can't manage to find how to set a language different from english for the created user when app is created

**Comment by michelson** [comment]
I will take a look at this asap, thanks for letting me know.

**Comment by michelson** [comment]
@Nuxor3000 look at the `langs:` key on the lang file. there is a list of the allowed langs that will be displayed on the lang chooser in the agent panel. Since the "IT" lang is not yet complete it is not enabled on the list. Langs with at least 90% completion will be added to that list.

Are you trying to display the lang on the agent panel or the webchat?

**Comment by Nuxor3000** [comment]
Yes I added it to the langs to use it even if with a not completely translated version.
I'm displaying the lang on both and it's not that bad.

To detail a bit the problem to help in finding the source of the problem :

In "login form" or in "accept invitation form", 
once "import I18n from '../../shared/FakeI18n';" added at the top of the file,
 I can use localized text but when an error occur 
(or as soon as I type something for the "accept invitation form"), 
the locale gets back to english.

**Comment by michelson** [comment]
Uhm... could you provide a screen recording?

**Comment by Nuxor3000** [comment]
https://user-images.githubusercontent.com/72187088/151217270-181d1614-7b89-4707-8bfa-520dd4a526a3.mov
